packages/terra-image/src/Image.jsx

```diff
@@ -143,24 +143,31 @@ class Image extends React.Component {
       src, variant, isFluid, alt, placeholder, height, width, onLoad, onError, fit, ...customProps
     } = this.props;
 
-    const imageClasses = cx([
+    const imageClassAttributes = [
       'image',
       fit,
       variant,
       customProps.className,
       { fluid: isFluid },
-    ]);
```

Comment:
Can you add like:

```diff
   const imageClasses = cx([
      'image',
      fit,
      variant,
      customProps.className,
      { fluid: isFluid },
+     { hidden: placeholder && this.state.isLoading }
    ]);

```

Reply:
I had something like this initially but decided against it as I didn't want to collide with the custom attributes a consumer could provide after the image had loaded.

Reply:
Let me know if you think this is a valid concern or not. From the way our team uses terra-image/profile-image it wouldn't affect us at all, and I'm sure anyone applying a hidden attribute to terra-image for some odd use case could find a work around.

Reply:
Nevermind, didn't know classNames removes falsy values.

Reply:
yeah it removes falsey values! It's great 😄